api/src/main/java/org/openmrs/BaseReferenceRangeObject.java

```diff
@@ -0,0 +1,326 @@
+	 * Returns high normal value of the referenceRange
+	 *
+	 * @return hiNormal the high normal value
+	 */
+	public Double getHiNormal() {
+		return this.hiNormal;
+	}
+
+	/**
+	 * Sets high normal value of the referenceRange
+	 *
+	 *  @param hiNormal low critical value to set
+	 */
+	public void setHiNormal(Double hiNormal) {
+		this.hiNormal = hiNormal;
+	}
+
+	/**
+	 * Returns low absolute value of the referenceRange
+	 *
+	 * @return lowAbsolute the low absolute value
+	 */
+	public Double getLowAbsolute() {
+		return this.lowAbsolute;
+	}
+
+	/**
+	 * Sets low absolute value of the referenceRange
+	 *
+	 * @param lowAbsolute low critical value to set
+	 */
+	public void setLowAbsolute(Double lowAbsolute) {
+		this.lowAbsolute = lowAbsolute;
+	}
+
+	/**
+	 * Returns low critical value of the referenceRange
+	 *
+	 * @return lowCritical the low critical value
+	 */
+	public Double getLowCritical() {
+		return this.lowCritical;
+	}
+
+	/**
+	 * Sets low critical value of the referenceRange
+	 *
+	 * @param lowCritical low critical value to set
+	 */
+	public void setLowCritical(Double lowCritical) {
+		this.lowCritical = lowCritical;
+	}
+
+	/**
+	 * Returns low absolute value of the referenceRange
+	 *
+	 * @return lowAbsolute the low absolute value
+	 */
+	public Double getLowNormal() {
+		return this.lowNormal;
+	}
+
+	/**
+	 * Sets low normal value of the referenceRange
+	 *
+	 * @param lowNormal low normal value to set
+	 */
+	public void setLowNormal(Double lowNormal) {
+		this.lowNormal = lowNormal;
+	}
+
+	/**
+	 * @return Returns the changedBy.
+	 */
+	@Override
+	public User getChangedBy() {
+		return changedBy;
+	}
+
+	/**
+	 * @param changedBy The changedBy to set.
+	 */
+	@Override
+	public void setChangedBy(User changedBy) {
+		this.changedBy = changedBy;
+	}
+
+	/**
+	 * @return Returns the dateChanged.
+	 */
+	@Override
+	public Date getDateChanged() {
+		return dateChanged;
+	}
+
+	/**
+	 * @param dateChanged The dateChanged to set.
+	 */
+	@Override
+	public void setDateChanged(Date dateChanged) {
+		this.dateChanged = dateChanged;
+	}
+
+	/**
+	 * @return Returns the creator.
+	 */
+	@Override
+	public User getCreator() {
+		return creator;
+	}
+
+	/**
+	 * @param creator The creator to set.
+	 */
+	@Override
+	public void setCreator(User creator) {
+		this.creator = creator;
+	}
+
+
+	/**
+	 * @return Returns the dateCreated.
+	 */
+	@Override
+	public Date getDateCreated() {
+		return dateCreated;
+	}
+
+	/**
+	 * @param dateCreated The dateCreated to set.
+	 */
+	@Override
+	public void setDateCreated(Date dateCreated) {
+		this.dateCreated = dateCreated;
+	}
+	
+	/**
+	 * @return Returns the retired.
+	 *
+	 * @deprecated as of 2.0, use {@link #getRetired()}
+	 */
+	@Override
+	@Deprecated
+	@JsonIgnore
+	public Boolean isRetired() {
+		return getRetired();
+	}
+
+	/**
+	 * This method delegates to {@link #isRetired()}.
+	 *
+	 * @see org.openmrs.Retireable#isRetired()
+	 */
+	@Override
+	public Boolean getRetired() {
```

Comment:
We do not retire or change obs. So this would not be a good base class for obs reference range